src/classes/CRLP_Batch_Base.cls

```diff
@@ -655,17 +655,14 @@ public abstract class CRLP_Batch_Base {
      * @param className The name of the skipped batch class to be used in the error message
      */
     protected void logDuplicateBatchError(String className) {
-
-        CRLPBatchException ex = new CRLPBatchException();
-        ex.setMessage(String.format(Label.CRLP_BatchSkipped, new List<String>{className}));
-        ERR_Handler.processError(ex, ERR_Handler_API.Context.CRLP);
-
+	    CRLPBatchException ex = new CRLPBatchException(); 
```

Comment:
Please fix indentation.

Reply:
I think I was the one who suggested moving the `logDuplicateBatchError()` method to the `UTIL_BatchJobService` class to avoid the code duplication.